### PR TITLE
Relocate heap dumps and error logs to ephemeral storage

### PR DESCRIPTION
**Pull request by eherot**

### CODE REVIEW

**Comment by eherot** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;AcoNewIM&#x2F;1190-0-5-move-cassandra-heap-dumps-off-of"> (0.5) Move Cassandra heap dumps off of &#x2F;</a>

File: files/default/cassandra-init [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+{
+    # Return
+    #   0 if daemon has been started
+    #   1 if daemon was already running
+    #   2 if daemon could not be started
+
+    ulimit -l unlimited
+    ulimit -n "$FD_LIMIT"
+
+    cassandra_home=`getent passwd cassandra | awk -F ':' '{ print $6; }'`
```
Comment (jeffbyrnes) [inline files/default/cassandra-init]: Why are there two vars like this? `$CASSANDRA_HOME` and `$cassandra_home`? If this is just how the script came, then definitely leave it alone.

File: test/integration/helpers/serverspec/default_installation.rb [hunk 2]
```diff
@@ -37,6 +39,10 @@
       expect(service('cassandra')).to be_running
     end
 
+    describe process 'java' do
+      its(:args) { should match %r{-XX:HeapDumpPath=#{log_dir}/cassandra/java_.*\.hprof} }
+    end
```
Comment (jeffbyrnes) [inline test/integration/helpers/serverspec/default_installation.rb]: Love this assertion.

File: files/default/cassandra-init (hunk 1 above)
Comment (eherot) [inline files/default/cassandra-init]: Yeah that's stock BS.

Comment (jeffbyrnes) [inline files/default/cassandra-init]: Alright, just leave it. Best not to stray too far from what they’re providing.

File: test/integration/helpers/serverspec/default_installation.rb (hunk 2 above)
Comment (eherot) [inline test/integration/helpers/serverspec/default_installation.rb]: I try to make it a habit to always add a test for the exact thing I just logged in to the test host in order to look at.

**Pull request merged**